kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiChannel.java

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
```

Comment:
I believe that the correct copyright header for the newly created files that you are contributing is something like:
```
/*******************************************************************************
  * Copyright (c) 2021 Sterwen-Technology and others
  * 
  * This program and the accompanying materials are made
  * available under the terms of the Eclipse Public License 2.0
  * which is available at https://www.eclipse.org/legal/epl-2.0/
  * 
  * SPDX-License-Identifier: EPL-2.0
  * 
  * Contributors:
  *  Sterwen-Technology
  ******************************************************************************/
```